### PR TITLE
Add release notes and document env var for Enterprise 3.7

### PR DESCRIPTION
**Pull request by hillaryfraley**
## Description

- Adds 3.7.0 release notes
- Adds CMS_OCCUPANCY_FRACTION to https://docs.sensu.io/sensu-enterprise/3.7/configuration/#sensu-enterprise-environment-variables

## Motivation and Context
Support Enterprise 3.7.0 release

## Review Instructions
Need to confirm addition of env var

### CODE REVIEW

File: content/sensu-enterprise/3.7/configuration.md [hunk 1]
```diff
@@ -18,10 +18,18 @@ sensu-enterprise service must be restarted before the new values can take effect
...
 ## Sensu Enterprise environment variables
 
-The Sensu Enterprise honors the following environment variables. For
+Sensu Enterprise honors the following environment variables. For
 configuration honored by both Sensu Enterprise and Sensu Core, see the
 [Sensu configuration reference documentation][1].
 
+CMS_OCCUPANCY_FRACTION   | 
+-------------------------|-------
+description              | Tunes the CMSInitiatingOccupancyFraction.
```
Comment (cwjohnston) [inline content/sensu-enterprise/3.7/configuration.md]:
```suggestion
description              | Configures the Initiating Occupancy Fraction threshold for the Java [Concurrent Mark Sweep (CMS) garbage collector][2] used by Sensu Enterprise.
```

File: content/sensu-enterprise/3.7/changelog.md [hunk 2]
```diff
@@ -59,6 +60,20 @@ _NOTE: Sensu Enterprise is built on Sensu Core. Sensu Core changes are documente
...
+released and is available for immediate download. Please note the
+following improvements:
+
+### CHANGES {#enterprise-v3-7-0-changes}
+
+- **BUGFIX**: Fixed Travis CI builds.
+
+Built on [Sensu Core 1.9.0][core-v1-9-0]:
+
+- **IMPROVEMENT**: You can now tune CMSInitiatingOccupancyFraction via an environment variable (e.g. `export CMS_OCCUPANCY_FRACTION="50"`).
```
Comment (cwjohnston) [inline content/sensu-enterprise/3.7/changelog.md]:
```suggestion
- **IMPROVEMENT**: You can now configure the threshold for Java CMSInitiatingOccupancyFraction garbage collection strategy via an environment variable (e.g. `export CMS_OCCUPANCY_FRACTION="50"`).
```

**Review by cwjohnston — CHANGES_REQUESTED** [review]
@amdprophet will you please review my suggestions here for accurate wording?

File: content/sensu-enterprise/3.7/changelog.md [hunk 3]
```diff
@@ -59,6 +60,20 @@ _NOTE: Sensu Enterprise is built on Sensu Core. Sensu Core changes are documente
...
 
+## Enterprise 3.7.0 Release Notes {#enterprise-v3-7-0}
+
+**December 19, 2019** &mdash; Sensu Enterprise version 3.7.0 has been
+released and is available for immediate download. Please note the
+following improvements:
+
+### CHANGES {#enterprise-v3-7-0-changes}
+
+- **BUGFIX**: Fixed Travis CI builds.
```
Comment (hillaryfraley) [inline content/sensu-enterprise/3.7/changelog.md]:
```suggestion
```

File: content/sensu-enterprise/3.7/changelog.md [hunk 4]
```diff
@@ -59,6 +60,19 @@ _NOTE: Sensu Enterprise is built on Sensu Core. Sensu Core changes are documente
...
 - [Enterprise 1.13.0 Release Notes](#enterprise-v1-13-0)
 
+## Enterprise 3.7.0 Release Notes {#enterprise-v3-7-0}
+
+**December 19, 2019** &mdash; Sensu Enterprise version 3.7.0 has been
+released and is available for immediate download. Please note the
+following improvements:
+
+### CHANGES {#enterprise-v3-7-0-changes}
+
```
Comment (hillaryfraley) [inline content/sensu-enterprise/3.7/changelog.md:70]:
```suggestion
```

File: content/sensu-enterprise/3.7/configuration.md (hunk 1 above)
Comment (cwjohnston) [reply content/sensu-enterprise/3.7/configuration.md]: Need to add `[2]: https://docs.oracle.com/javase/8/docs/technotes/guides/vm/gctuning/cms.html` at the bottom for this link ref

File: content/sensu-enterprise/3.7/changelog.md [hunk 5]
```diff
@@ -59,6 +60,18 @@ _NOTE: Sensu Enterprise is built on Sensu Core. Sensu Core changes are documente
...
+Built on [Sensu Core 1.9.0][core-v1-9-0]:
+
+- **IMPROVEMENT**: You can now configure the threshold for Java CMSInitiatingOccupancyFraction garbage collection strategy via an environment variable (e.g. `export CMS_OCCUPANCY_FRACTION="50"`).
```
Comment (cwjohnston) [inline content/sensu-enterprise/3.7/changelog.md]:
```suggestion
- **IMPROVEMENT**: You can now configure the Initiating Occupancy Fraction threshold for Java garbage collection via an environment variable (e.g. `export CMS_OCCUPANCY_FRACTION="50"`).
```

**Review by cwjohnston — CHANGES_REQUESTED** [review]

**Review by cwjohnston — APPROVED** [review]

File: content/sensu-enterprise/3.7/changelog.md [hunk 6]
```diff
@@ -59,6 +60,18 @@ _NOTE: Sensu Enterprise is built on Sensu Core. Sensu Core changes are documente
 - [Enterprise 1.14.0 Release Notes](#enterprise-v1-14-0)
 - [Enterprise 1.13.0 Release Notes](#enterprise-v1-13-0)
 
+## Enterprise 3.7.0 Release Notes {#enterprise-v3-7-0}
+
+**December 19, 2019** &mdash; Sensu Enterprise version 3.7.0 has been
```
Comment (cwjohnston) [inline content/sensu-enterprise/3.7/changelog.md]:
```suggestion
**January 7, 2020** &mdash; Sensu Enterprise version 3.7.0 has been
```

**Pull request merged**